src/main/java/org/spongepowered/common/data/processor/data/block/TreeDataProcessor.java

```diff
@@ -45,7 +45,10 @@ public TreeDataProcessor() {
         super(Keys.TREE_TYPE, input -> input.getItem() == ItemTypes.PLANKS || input.getItem() == ItemTypes.LEAVES
                 || input.getItem() == ItemTypes.LEAVES2 || input.getItem() == ItemTypes.LOG
                 || input.getItem() == ItemTypes.LOG2 || input.getItem() == ItemTypes.SAPLING
-                || input.getItem() == ItemTypes.WOODEN_SLAB);
+                || input.getItem() == ItemTypes.WOODEN_SLAB || input.getItem() == ItemTypes.BOAT
```

Comment:
maybe move the second argument of this to a private static method to make this nicer